arch/cortex-m/src/systick.rs

```diff
@@ -142,6 +149,12 @@ impl kernel::SysTick for SysTick {
         SYSTICK_BASE.syst_cvr.set(0);
     }
 
+    fn pause(&self) {
```

Comment:
I don't think RISC-V can pause it's counter, so I'm not sure how this would work in RV32.

Reply:
looks like you are right, I will need to rework this :/ it was so elegant for arm..